public/app/features/plugins/cdn/utils.ts

```diff
@@ -7,19 +7,23 @@ export function transformPluginSourceForCDN({
   url,
   source,
   transformSourceMapURL = false,
+  transformAssets = true,
 }: {
   url: string;
   source: string;
   transformSourceMapURL?: boolean;
+  transformAssets?: boolean;
```

Comment:
Nit: I guess it doesn't need to be an optional property if it has a default value?

Reply:
If you don't make it optional it'll fail other places where you call it and not directly pass the parameter.

Reply:
Ahh, true, it's not like a function parameter with a default value.